src/tests/bignum_test.nr

```diff
@@ -921,3 +921,56 @@ fn test_sub_underflow_regression() {
     let c = b - a;
     assert(c.limbs == [0, 0, 0]);
 }
+
+#[test]
+fn test_cmp_BN() {
```

Comment:
Can you move these all inside of a submodule grouping all of the tests related to `cmp`?